### PR TITLE
IUM-1930 updated/created markdown files

### PR DESCRIPTION
**Pull request by TimLawton-Auth0**
By submitting a PR to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

Adding and updating the README.md, CODEOWNERS.md, and CONTRIBUTING.md files into this repository as part of the repository ownership task.

### References

WIKI: https://auth0team.atlassian.net/wiki/spaces/ENG/blog/2022/01/12/2655977953/Repository+Ownership
JIRA: https://auth0team.atlassian.net/browse/IUM-1930?atlOrigin=eyJpIjoiYWNkYmQyNzdjZDA3NGJjMmJiMjFmODZjOTg2Zjk5MDAiLCJwIjoiaiJ9

### Testing

This can be tested by creating a PR to this repo. If successful, a reviewer will be automatically assigned.

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [x] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used, if not `master`

### CODE REVIEW

**Comment by bernardwolff** [comment]
This sentence in the readme seems to be cut off:
> Securely store and manage username / password credentials either in your own database

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Usage steps are located in README.md at the root of this repository
```
[the remark this answers is not in the document]
Comment (bernardwolff) [reply CONTRIBUTING.md:1]: The CONTRIBUTING.md file should probably contain some instructions for how to contribute, like where to add new script templates, how to test them, how to make sure they get deployed, etc. I've never worked in this repo but maybe you could ask in the team channel.

[the remark this answers is not in the document]
Comment (bernardwolff) [reply CONTRIBUTING.md]: This is a good example of a CONTRIBUTING file https://github.com/auth0/auth0-notifications/blob/master/CONTRIBUTING.md

[the remark this answers is not in the document]
Comment (TimLawton-Auth0) [reply CONTRIBUTING.md]: Changed CONTRIBUTING and README files

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Contributing to Auth0 projects
+
+A big welcome and thank you for considering contributing to the Auth0 open source projects. It’s people like you that make it a reality for users in our community.
```
Comment (madsharm) [inline CONTRIBUTING.md:3]: Looks like we are opening this repo for contributions. Do we have a plan on how to use team's bandwidth on this commitment.

Comment (TimLawton-Auth0) [reply CONTRIBUTING.md]: I updated the contributing file to redirect them into opening a ticket. I removed "submitting a pull request" section

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Code of Conduct
+
+By participating and contributing to this project, you are expected to uphold our [Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).
+
+## Getting Started
+
+### Making Changes
+
+When contributing to a repository, the first step is to open an issue in that repository to discuss the change you wish to make before making them.
```
Comment (madsharm) [inline CONTRIBUTING.md:25]: Do we need this? Do we want to recommend opening Github issue as well as support ticket?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+### Opening Issues
+
+Before you submit a new issue please make sure to search all open and closed issues. It is possible your feature request/issue has already been answered.
```
Comment (madsharm) [inline CONTRIBUTING.md:29]: Not sure if people can search issues in support ticket system. Or do we need Github issues here?

Comment (TimLawton-Auth0) [reply CONTRIBUTING.md]: I believe that our Github issues is unmonitored/unused. I rewrote the file to direct a would-be-contributor to submit a service ticket.

File: CONTRIBUTING.md (hunk 3 above)
Comment (TimLawton-Auth0) [reply CONTRIBUTING.md]: I removed this.

**Review by madsharm — APPROVED** [review]

**Review by vmarceau — APPROVED** [review]

**Pull request merged**